### ISSUE TITLE
CI/CD: Setup Deployment to Production on Merge to Master Branch

### ISSUE THREAD

**Issue by AleksandrMolchagin** [AmericaSCORESBayArea/salesforce-data-api#234]
**Description:**

We need to configure our CI/CD pipeline to automatically deploy the application to the production environment whenever changes are merged into the master branch. The pipeline will be similar to the Sandbox pipeline, but `.jar` files should be added to the releases in GitHub.

Depends on #233.
